### ISSUE TITLE
Simplify inputs for common symbols/types

### ISSUE THREAD

**Issue by DennisSmolek** [pmndrs/lamina#10]
Consider this blend effect
```js
        <LayerMaterial transparent depthWrite={false} side={THREE.FrontSide} blending={THREE.AdditiveBlending}>
```
With strings:
```js
        <LayerMaterial transparent depthWrite={false} side="front" blending="additive">
```

Side has 3 options, front, back, both. Blending does have a lot more, so you could still allow it to be passed, but otherwise make it easier...

**Comment by FarazzShaikh** [comment]
I agree this is better to look at. Currently `LayerMaterial` accepts all `ShaderMaterial` props. I just think it would break consistency with other Materials in the library 

For blending, isn't it possible to define a custom blending function in that prop?

**Comment by FarazzShaikh** [comment]
Closed via `1.1.1`
